dashboard/src/pages/Repository.tsx

```diff
@@ -55,7 +55,7 @@ function RepositoryPage() {
         onRepositoryChanged={handleRepositoryChanged}
         selectedRepository={selectedRepository}
       />
-      <div className="w-10/12">
+      <div className="w-10/12" style={{width: "1014px"}}>
```

Comment:
what the hell?
you have tailwind! and we already use it! w-10/12
don't use inline styles!